index.html

```diff
@@ -1,18 +1,21 @@
 <!doctype html>
 <html lang="en">
-  <head>
-    <meta charset="UTF-8" />
-    <link rel="icon" type="image/x-icon" href="/favicon.ico" />
-    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
-    <link rel="manifest" href="/webmanifest.json" />
-    <!-- Runtime environment variables -->
-    <!-- With Vite, type must be text/javascript, see:
+
+<head>
```

Comment:
This is formatting change that should not be here.

Reply:
my bad, i'll try to fix it.

Reply:
@blcham i have only  printWidth": 120 and this is correct, right? i'll check how can i configure my IDE